azure-pipelines.yml

```diff
@@ -5,32 +5,146 @@ trigger:
       - "*"
 
 variables:
-- name: SolutionBaseName
-  value: SFA.DAS.Charities.Api
+- group: Release Management Resources
+- group: RELEASE das-charities-api
 - name: BuildConfiguration
   value: release
-- name: BuildPlatform
-  value: any cpu
-
-pool:
-  name: DAS - Continuous Integration
-  workspace:
-    clean: all
+- name: SolutionBaseName
+  value: SFA.DAS.Charities.Api
+- name: DACPACSolutionBaseName
+  value: SFA.DAS.Charities.Database
 
 resources:
   repositories:
   - repository: self
   - repository: das-platform-building-blocks
     type: github
     name: SkillsFundingAgency/das-platform-building-blocks
-    ref: refs/tags/0.3.31
+    ref: refs/tags/0.4.20
     endpoint: SkillsFundingAgency
-
+  - repository: das-platform-automation
+    type: github
+    name: SkillsFundingAgency/das-platform-automation
+    ref: refs/tags/4.5.3
+    endpoint: SkillsFundingAgency
+  pipelines:
+  - pipeline: das-employer-config
+    project: Digital Apprenticeship Service
+    source: das-employer-config
+    branch: master
 stages:
 - stage: Build
   jobs:
   - template: pipeline-templates/job/code-build.yml
     parameters:
       SolutionBaseName: $(SolutionBaseName)
       BuildConfiguration: $(BuildConfiguration)
-    
\ No newline at end of file
+  - template: azure-pipelines-templates/build/job/dacpac-template.yml@das-platform-building-blocks
+    parameters:
+      SolutionBaseName: $(DACPACSolutionBaseName)
+      SqlProjectPath: src/$(DACPACSolutionBaseName)/$(DACPACSolutionBaseName).sqlproj
+
+- stage: Deploy_AT
+  dependsOn: Build
+  displayName: Deploy to AT
+  variables:
+  - group: DevTest Management Resources
+  - group: AT DevTest Shared Resources
+  #- group: AT das-charities-api
```

Comment:
Can these be deleted?

Reply:
Just confirmed and no secrets will be involved so will delete the env specific variable groups